src/wilson/dom.cljs

```diff
@@ -78,6 +79,38 @@
   ([ks rows]
    (table ks rows {})))
 
+
+(defn sorted-table
+  ([ks rows opts]
+   (let [state (r/atom {:sort-key (first ks)
+                        :sort-order :asc})
+         state-deref @state
+         sort-fns (or (:sort-fns opts)
```

Comment:
You should document what sort-fns is, because I'm not sure I understand how to use it.